build.sbt

```diff
@@ -137,10 +139,8 @@ lazy val V = new {
   // List of supported Scala versions in SemanticDB. Needs to be manually updated
   // for every SemanticDB upgrade.
   def supportedScalaVersions =
-    Seq("2.12.8", "2.12.7", "2.11.12") ++ deprecatedScalaVersions
-  def deprecatedScalaVersions = Seq[String](
-    "2.12.6", "2.12.5", "2.12.4", "2.11.11", "2.11.10", "2.11.9"
-  )
+    Seq("2.13.0-RC1", "2.12.8", "2.12.7", "2.11.12")
+  def deprecatedScalaVersions = Seq[String]()
```

Comment:
This is completely arbitrary, but I think it might be a good time to drop the deprecated Scala versions. More concretely, we need Scalameta 4.1.11, which is not published for the deprecated versions anymore.